README.md

```diff
@@ -8,16 +8,17 @@ This repository contains samples for usage of the Google Cloud Machine Learning
 
 ### CPU and GPU
 
-* [Keras Census](census/keras) - Keras single node model
-* [Census](census) - TensorFlow census models
-  * [Canned Estimator](census/estimator) - TensorFlow canned estimator model
-  * [Custom Estimator](census/customestimator) - TensorFlow custom estimator model
+* [Census](census) - Predict a person's income level
+  * [Keras Census](census/keras) - [Keras](https://keras.io/) single node model
+  * [Canned Estimator](census/estimator) - [TensorFlow canned estimator](https://www.tensorflow.org/programmers_guide/estimators#pre-made_estimators) model
+  * [Custom Estimator](census/customestimator) - [TensorFlow custom estimator](https://www.tensorflow.org/programmers_guide/estimators#custom_estimators) model
   * [Low Level TF](census/tensorflowcore) - TensorFlow low level API model
-* [Criteo](criteo_tft) - Advertisement CTR prediction on Criteo dataset
-* [Flowers](flowers) - Image transfer learning using flowers dataset
-* [Movielens](movielens) - Personalized recommendation using movielens dataset
-* [Reddit](reddit_tft) - Wide and Deep model using reddit dataset
-* [Cifar10](https://github.com/tensorflow/models/tree/master/tutorials/image/cifar10_estimator) - Multi-GPU Cifar10
+* [Criteo](criteo_tft) - Predict how likely a person is to click on an
```

Comment:
Its a classification problem. May be rephrase:
"Predict how likely a person is to click on an.." => "Predict if a user will click on an Ad"

Reply:
It is a soft classification and so the outputs should be interpreted as probabilities of belonging to one of the classes. The text as it stands is a more accurate representation of this.

Reply:
What do you mean soft classification?

Reply:
It is a probability of belonging to the "clicked" class that is returned as output - at least that's what I remembered from last time I tested the model.